tests/vaults/VaultOps.test.ts

```diff
@@ -476,7 +476,7 @@ describe('VaultOps', () => {
       expect(secretList).toHaveLength(0);
     });
   });
-  // Not sure if hidden file names are a special case but I'm keeping the tests just in case
+  // Not sure if hidden file names are a special case, but I'm keeping the tests just in case
   test('adding hidden files and directories', async () => {
```

Comment:
Let's actually test this, and remove the comment.

Reply:
Comment has been removed.